app/components/publications/publication/translations/documents-edit-modal.js

```diff
@@ -0,0 +1,48 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+
+export default class PublicationsTranslationDocumentEditModalComponent extends Component {
+  @tracked name = this.args.sourceDocument.name;
+  @tracked pagesAmount = this.args.sourceDocument.pages;
+  @tracked wordsAmount = this.args.sourceDocument.words;
+  @tracked isSourceForProofPrint;
+
+  constructor() {
+    super(...arguments);
+    this.loadData.perform();
+  }
+
+  @task
+  *loadData() {
+    const subcase = yield this.args.sourceDocument.publicationSubcase;
+    this.isSourceForProofPrint = !!subcase ;
+  }
+
+  get saveIsDisabled() {
+    return this.name === null;
+  }
```

Comment:
Ik snap wat je hiermee wilt doen, maar als je een name helemaal zou deleten, heb je dan niet eerder een lege string

Reply:
Vermoedelijk wil je eerder met iets in de richting van https://api.emberjs.com/ember/3.18/functions/@ember%2Futils/isEmpty werken ...